README.md

```diff
@@ -1 +1,45 @@
-# `echo-server`
\ No newline at end of file
+# Echo Server
+Push server for the WalletConnect v2 Protocol
+
+> **Note** This is only available for WalletConnect v2 and is a breaking change from the Push supported in v1.
+
+## Supporting Notifications
+> **Note** Full documentation will be available soon. This is only a brief overview.
```

Comment:
What do you plan on adding to the full documentation?

Reply:
The API will be fully documented as well as SDK docs. This is just a quick overview of the spec for people to start supporting them.